libs/transfer/src/lib/services/make-transfer-permissions.service.spec.ts

```diff
@@ -1,21 +1,20 @@
-import { PropertyResolver } from '@backbase/foundation-ang/web-sdk';
 import { MakeTransferPermissionsService } from './make-transfer-permissions.service';
 
 describe('MakeTransferPermissionsService', () => {
   let service: MakeTransferPermissionsService;
-  let conditionsServiceStub: PropertyResolver;
+  const conditionsServiceStub: any = {
```

Comment:
Why using `any` ?

Reply:
Do you know of a typing solution that would fix this. In Jest with Angular, I read to create the component as below. From my experience Jest is more developer friendly and significantly faster, however this is no createSpyObj function. Without createSpyObj, I wasn't sure how to get the typing correct for for creating the component. I tried combinations of partials, pick, and add.

 ```
   component = new TransactionsViewComponent(
      mockActivatedRoute,
      mockTransactionViewService,
      mockTransactionsCommunicationService
    );
```

Reply:
I had watched a video testing this way using Jest in Angular, if there is a way to use a type I agree it would be better.

Reply:
Before your changes the previous test was using an interface. Check if it can still be used in yours.

Reply:
I found a way to use better typing, I will change them all today.

Reply:
Something like this could work. If you like this approach better I can update them all
```
  const snapshot: Partial<ActivatedRouteSnapshot> = {
    data: {
      title: 'someTitle',
    },
  };
  let scheduler: TestScheduler;
  let component: TransactionsViewComponent;
  const mockActivatedRoute = {
    snapshot: snapshot as ActivatedRouteSnapshot,
  };
  const mockTransactionsHttpService: Partial<TransactionsHttpService> = {
    transactions$: of(transactionsMock),
  };
  let mockTransactionsCommunicationService:
    | Partial<TransactionsCommunicationService>
    | undefined = {
    latestTransaction$: of(debitMockTransaction),
  };
  const createComponent = () => {
    component = new TransactionsViewComponent(
      mockActivatedRoute as ActivatedRoute,
      mockTransactionsHttpService as TransactionsHttpService,
      mockTransactionsCommunicationService as TransactionsCommunicationService
    );
  };
  beforeEach(() => {
    createComponent();
  });
```

Reply:
Admittedly I have also used `any` for these cases in the past sometimes, however I think adding type safety with `Partial` is preferable. So changing a method in a service or component breaks test in compile time, instead of run time.

I'm curious about other opinions about removing Testbed. Personally I prefer testing without, because of performance, and it feels you are testing closer to "reality", for me personally. However Testbed seems to be the standard way of testing Angular components AFAIK, so not sure if what example we want to give others.

Reply:
@jordydejong I will make these changes today.